packages/nimble-components/src/rich-text/editor/styles.ts

```diff
@@ -188,6 +194,24 @@ export const styles = css`
         color: ${controlLabelDisabledFontColor};
     }
 
+    nimble-anchor {
+        white-space: normal;
+        ${
+            /**
+             * Restricting the pointer events for the following reasons:
+             * 1. nimble-anchor inside a "contenteditable" div is not working as native HTML anchor tag.
+             *    i.e. clicking on the link opens in the same tab whereas the default behavior of native HTML anchor
+             *    tag is not clickable inside "contenteditable" div.
+             *    Issue link: https://github.com/ni/nimble/issues/1502
+             * 2. Restricting the user from opening a link using the right-click context menu: If the user manually edits
+             *    the link's text content, the 'href' attribute of the anchor tag will not be updated. If they attempt to open it using
```

Comment:
Since there might be any number of links in the editor, we couldn't find a better way to update the `href` of the current link. Even the default linking behavior in Tiptap doesn't support updating the link. 
![absolute-link-tiptap](https://github.com/ni/nimble/assets/123377523/93c3fbfa-b222-441d-bf57-94ea7552c4ba)

When I am looking into some of the other WYSIWYG editors, they usually don't update the `href` when updating the text content. As the other editors might be supporting the link markdown format `[text](https://foo)`. So, until we enable `link` in markdown, we might have to have this workaround.